dev-tools/ecs-migration.yml

```diff
@@ -436,3 +436,8 @@
   to: user_agent.original
   alias: true
   copy_to: false
+
+- from: monitor.url
```

Comment:
Is this WIP? I don't see it changed in the code.

Reply:
I had been hoping to do two in one in this PR, but I think you're right, let's do one per PR